### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by sharonwarner**
- Updates content edits from the active google doc

### CODE REVIEW

File: src/client/components/BPOButton/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ function BPOButton() {
       onClick={() => logEvent("register-or-login")}
       target="_blank"
     >
-      Register or log in at Benefit Programs Online
+      Go to Benefits Programs Online
```
Comment (kalvinwang) [inline src/client/components/BPOButton/index.js:13]: I thought it was Benefit? Regardless, it's different in two different places in this PR so only one can be right

**Comment by mkhandekar** [comment]
Adding the document we worked from for this content update! 
[CA UI application guidance - content – ACTIVE DRAFT (2).docx](https://github.com/cagov/benefits-screener/files/4566362/CA.UI.application.guidance.-.content.ACTIVE.DRAFT.2.docx)

File: src/client/components/TabPaneContent0/index.js [hunk 2]
```diff
@@ -23,63 +23,98 @@ function TabPaneContent0({ loadTab, tabSlugs, getTabTitle }) {
...
-                on the Employment History screen when you supply your last
-                employer information, select <strong>No</strong>.
+                On the Employment History screen when you supply your last
+                employer information, select <strong>"No"</strong>.
               </li>
               <li>
-                on the Availability Information page, answer question 7 with{" "}
-                <strong>No</strong>.
+                On the Availability Information page, answer question 7 with{" "}
+                <strong>"No"</strong>.
```
Comment (mkhandekar) [inline src/client/components/TabPaneContent0/index.js:52]: These don't have to be bolded.

File: src/client/components/BPOButton/index.js (hunk 1 above)
Comment (mkhandekar) [reply src/client/components/BPOButton/index.js:13]: It is "Benefit Programs Online"

**Pull request merged**